packages/lesswrong/lib/collections/doppelCommentVotes/collection.ts

```diff
@@ -0,0 +1,18 @@
+import { addUniversalFields, getDefaultMutations, getDefaultResolvers } from "@/lib/collectionUtils";
+import { createCollection } from "@/lib/vulcan-lib";
+import schema from "./schema";
+
+const DoppelCommentVotes: DoppelCommentVotesCollection = createCollection({
+  collectionName: "DoppelCommentVotes",
+  typeName: "DoppelCommentVote",
+  schema,
+  logChanges: true,
+  resolvers: getDefaultResolvers('DoppelCommentVotes'),
+  mutations: getDefaultMutations('DoppelCommentVotes'),
+});
+
+addUniversalFields({
+  collection: DoppelCommentVotes,
+});
+
```

Comment:
Oh, plausibly we want a unique index on (userId, commentId), here?  (Assuming we do want to restrict users to a single vote per comment + doppelComments.)